WordPress/gutenberg
# [RNMobile] Ensure that `blockType` is defined when accessing `wrapperProps`

## What?
Ensure that `blockType` is defined when accessing `wrapperProps` in block.native.js. 

Fixes:
* https://github.com/wordpress-mobile/gutenberg-mobile/issues/6410

## Why?
Prevents a crash in certain circumstances when `blockType` is not defined, like when the block has been unregistered.

## How?
Matches the web block.js behavior by adding a check to ensure `blockType` is defined when accessing `wrapperProps`.

**Current web behavior:**
https://github.com/WordPress/gutenberg/blob/550966a9d6eee2cb187e3bfcdc432ef4c14737fd/packages/block-editor/src/components/block-list/block.js#L151-L157

**Current mobile behavior:**
https://github.com/WordPress/gutenberg/blob/550966a9d6eee2cb187e3bfcdc432ef4c14737fd/packages/block-editor/src/components/block-list/block.native.js#L245-L249

**Proposed mobile change:**
https://github.com/WordPress/gutenberg/blob/4d2e453e4cf4fce1d956a6e16965a7d38405c6af/packages/block-editor/src/components/block-list/block.native.js#L245-L252


## Testing Instructions
* Open the app and go to the Pages screen.
* Create a blank page.
* Switch to HTML mode and copy the following HTML code:
```

```
* Switch back to Visual mode.
* Save the page.
* Close the editor and open the page again.
* Observe the app does not crash.
